code/modules/food/recipes_microwave.dm

```diff
@@ -903,3 +900,65 @@ I said no!
 	)
 	reagent_mix = RECIPE_REAGENT_REPLACE
 	result = /obj/item/weapon/reagent_containers/food/snacks/nugget
+
+// Chip update
+/datum/recipe/tortila
+	reagents = list("flour" = 5,"sodiumchloride" = 1,"water" = 5)
+	result = /obj/item/weapon/reagent_containers/food/snacks/tortilla
+	reagent_mix = RECIPE_REAGENT_REPLACE
+
+/datum/recipe/taconew
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/tortilla,
+		/obj/item/weapon/reagent_containers/food/snacks/cutlet,
+		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/taco
+
+/datum/recipe/taconew_alt
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/tortilla,
+		/obj/item/weapon/reagent_containers/food/snacks/meatball,
+		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/taco
+
+/datum/recipe/chips
+	reagents = list("sodiumchloride" = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/tortilla
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/chipplate
+	reagent_mix = RECIPE_REAGENT_SUM
+
+/datum/recipe/nachos
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/chipplate,
+		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/chipplate/nachos
+	reagent_mix = RECIPE_REAGENT_SUM
+
+/datum/recipe/salsa
+	fruit = list("chili" = 1, "tomato" = 1,"lime" = 1)
+	reagents = list("spacespice" = 1, "blackpepper" = 1,"sodiumchloride" = 1)
+	result = /obj/item/weapon/reagent_containers/food/snacks/dip/salsa
+	reagent_mix = RECIPE_REAGENT_SUM
+
+/datum/recipe/guac
+	fruit = list("chili" = 1,"lime" = 1)
+	reagents = list("spacespice" = 1, "blackpepper" = 1,"sodiumchloride" = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/tofu
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/dip/guac
+	reagent_mix = RECIPE_REAGENT_SUM
+
+/datum/recipe/cheesesauce
+	fruit = list("chili" = 1, "tomato" = 1)
+	reagents = list("spacespice" = 1, "blackpepper" = 1,"sodiumchloride" = 1)
+	items = (
```

Comment:
How the fuck does this even compile. 🤔 
This should be:
```DM
	items = list(
		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge
	)
```

Reply:
oh that explains why it wasn't passing travis ty papa